docs/howtos/modules/deadlines/pages/testing.adoc

```diff
@@ -0,0 +1,36 @@
+= Testing the deadline
+
+Testing behaviour related to deadlines is easy by using either the `SagaTestFixture` or the `AggregateTestFixture` these will include configuration to use a `StubDeadlineManager`.
```

Comment:
I think we can leave out the use of the `StubDeadlineManager` for testing here to simplify the how-to slightly.

Reply:
Yes, removed it.